src/navigation/components/Navigation.jsx

```diff
@@ -55,4 +55,4 @@ class Navigation extends Component {
     }
 }
 
-export default Navigation;
+export default Navigation;
```

Comment:
Please configure your editor to insert newlines. Reference [this](https://github.com/nsoss/srce-desktop-app/pull/27#discussion_r348381193) comment from a previous review for the rationale.

Reply:
This is still unresolved.